pentestportal_sg_rules.tf

```diff
@@ -0,0 +1,40 @@
+# Allow ingress from Guacamole instance via ssh
+# For: DevOps ssh access from Guacamole instance to pentest portal instance
+resource "aws_security_group_rule" "pentestportal_ingress_from_guacamole_via_ssh" {
+  provider = aws.provisionassessment
+
+  security_group_id = aws_security_group.pentestportal.id
+  type              = "ingress"
+  protocol          = "tcp"
+  cidr_blocks       = ["${aws_instance.guacamole.private_ip}/32"]
```

Comment:
I don't _think_ the quotes are necessary here.

Reply:
As of Terraform v0.12.29, it appears the quotes are needed:
```
Error: Invalid character
  on pentestportal_sg_rules.tf line 9, in resource "aws_security_group_rule" "pentestportal_ingress_from_guacamole_via_ssh":
   9:   cidr_blocks       = [${aws_instance.guacamole.private_ip}/32]

This character is not used within the language.

Error: Invalid expression
  on pentestportal_sg_rules.tf line 9, in resource "aws_security_group_rule" "pentestportal_ingress_from_guacamole_via_ssh":
   9:   cidr_blocks       = [${aws_instance.guacamole.private_ip}/32]

Expected the start of an expression, but found an invalid expression token.
```

Reply:
Got it.  I was wrong.